resources/views/components/link.blade.php

```diff
@@ -0,0 +1,17 @@
+{{-- It's important that this file does NOT have a newline at the end. --}}
```

Comment:
Fair, but it currently *has* a newline at the end

Reply:
Yep and it's added automatically by my freaking editor :-(

let me fix it

Reply:
Fixed